charts/linkerd2/templates/prometheus.yaml

```diff
@@ -224,6 +224,7 @@ spec:
         - mountPath: /etc/prometheus
           name: prometheus-config
           readOnly: true
+      {{- $_ := set .Values.global.proxy "resources" (default .Values.global.proxy.resources .Values.prometheusProxyResources) }}                  
```

Comment:
Same comment as above, but here you missed the dash at the end `-}}`

Reply:
Setting the the dashes as you propose would add the prefix spaces of each line, would it not?
I have to test that.

Reply:
Ah, I just noticed, I am not consistent. I will fix that.